content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -88,187 +134,148 @@ following:
-    using var activity = MyActivitySource.StartActivity("SayHello");
-    activity?.SetTag("foo", 1);
-    activity?.SetTag("bar", "Hello, World!");
-    activity?.SetTag("baz", new int[] { 1, 2, 3 });
+    public static Meter Meter = new(ServiceName);
+    public static Counter<long> RequestCounter =
+        Meter.CreateCounter<long>("Request Counter");
+}
 
-    // Up a counter for each request
-    counter.Add(1);
+```
+
+Now we can increment the counter in our `Index` action.
+
+```csharp
+    public IActionResult Index()
+    {
+        // do other stuff
 
-    return "Hello, World!";
-});
+        DiagnosticsConfig.RequestCounter.Add(1,
+            new("Action", nameof(Index)),
+            new("Controller", nameof(HomeController)));
 
-app.Run();
+        return View();
+    }
 ```
 
-The output will be similar as with tracing, but now includes the request counter
-and periodically exports a histogram of request times for each route.
+You'll notice here that we're also adding labels to our request counter that
+distinguish it from other requests. You should now see an output like this.
 
 <details>
 <summary>View example output</summary>
 
-```text
-Activity.TraceId:            bdf32b90913015106be97242fbfca649
-Activity.SpanId:             7dba8699cd7ae27c
-Activity.TraceFlags:         Recorded
-Activity.ParentSpanId:       c28199417929ae65
-Activity.ActivitySourceName: MyCompany.MyProduct.MyService
-Activity.DisplayName:        SayHello
-Activity.Kind:               Internal
-Activity.StartTime:          2022-11-16T06:09:37.7932050Z
-Activity.Duration:           00:00:00.0018200
-Activity.Tags:
-    foo: 1
-    bar: Hello, World!
-    baz: [1,2,3]
-Resource associated with Activity:
-    service.name: MyCompany.MyProduct.MyService
-    service.version: 1.0.0
-    service.instance.id: c1593668-d427-4e5d-a7a6-03c3dd4fa342
-
-Activity.TraceId:            bdf32b90913015106be97242fbfca649
-Activity.SpanId:             c28199417929ae65
-Activity.TraceFlags:         Recorded
-Activity.ActivitySourceName: OpenTelemetry.Instrumentation.AspNetCore
-Activity.DisplayName:        /hello
-Activity.Kind:               Server
-Activity.StartTime:          2022-11-16T06:09:37.7878590Z
-Activity.Duration:           00:00:00.0400410
-Activity.Tags:
-    http.host: localhost:7026
-    http.method: GET
-    http.scheme: https
-    http.target: /hello
-    http.url: https://localhost:7026/hello
-    http.flavor: 1.1
-    http.user_agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36
-    http.status_code: 200
-Resource associated with Activity:
-    service.name: MyCompany.MyProduct.MyService
-    service.version: 1.0.0
-    service.instance.id: c1593668-d427-4e5d-a7a6-03c3dd4fa342
-
-Export app.request-counter, Meter: MyCompany.MyProduct.MyService
-(2022-11-16T06:09:04.2555860Z, 2022-11-16T06:09:44.2393710Z] LongSum
+```
+Export app.request_counter, Meter: MyService
```

Comment:
This does not match the name of your counter. That being said I was curious how the console exporter rendered spaces and I ran into https://github.com/open-telemetry/opentelemetry-dotnet/issues/4252

Reply:
fixed the above line... this was part of the "doesn't allow spaces" issue.